jekyll/_cci2/using-orbs.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: classic-docs
+title: "Using Orbs"
+short-title: "Using Orbs"
+description: "Starting point for CircleCI Orbs"
+categories: [getting-started]
+order: 1
+---
+
+This document provides a basic example of using an inline Orb and elements of Orbs followed by a conceptual overview of Orbs design.
+
+* TOC
+{:toc}
+
+## Introduction
+
+Orbs are package systems (similar to Ruby Gems) that you can use to quickly and easily get up and running using the CircleCI platform. 
+
+To use an existing orb inline in your `config.yml` file, invoke it with the `orbs` key. The following example invokes the `hello-build` orb in the `circleci` namespace.
```

Comment:
the word "inline" here is not generally how we use that term -- this is importing a registered orb, whereas "inline" generally refers to when you define the actual contents of the orb as the value of the key under `orbs`

Reply:
thanks, removed